### PR TITLE
Create readmes for every component based on GitHub wiki content

### PR DESCRIPTION
**Pull request by Buuhuu**
## Description

To bring the documentation of the Email Core Components in line with the Core Components, all content about the individual components in the GitHub wiki were moved to readme files for each component.

Furthermore some previous changes to the example content got reverted to match the readmes and some parts in the readmes got removed to match the implementation.

## Related Issue

CQDOC-19486

## Motivation and Context

Standardizes documentation across components projects so users can easily find content. Removes redundant and outdated content from GitHub wiki

## How Has This Been Tested?

No. But only readme files were added.

## Screenshots (if appropriate):

n/a

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: content/src/content/jcr_root/apps/core/email/components/button/v1/button/readme.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+The Button Component serves as a call-to-action link.
+
+## Features
+
+* Supports editable templates
+* Title, description, tagging, and language definition
+* HTML ID
+* Cloud service configuration
+* Email subject, pre-header, plain text, and reference URL
```
Comment (bohnertchris) [inline content/src/content/jcr_root/apps/core/email/components/button/v1/button/readme.md:28]: I think this is from the Page component and should probably be removed.

File: content/src/content/jcr_root/apps/core/email/components/image/v1/image/readme.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+# Image Component
+
+Extensible image component for the Core Email Components for composing campaign content written in HTL and based on the [Image Core Component](https://github.com/adobe/aem-core-wcm-components/tree/main/content/src/content/jcr_root/apps/core/wcm/components/image/v3/image)
+
+The Image Component allows embedding images in campaign content.
+
+## Features
+
+* Linkable to content pages, external URLs or page anchors
+* Allows an icon identifier to be configured for rendering an icon
```
Comment (bohnertchris) [inline content/src/content/jcr_root/apps/core/email/components/image/v1/image/readme.md:25]: How does this work? In the UI, there is no indication of this feature. If it is considered "advanced" or only for developers, we need to document it more clearly here. If It is expected that the author user can use this feature, we will need to include it in the Experience League docs as well.

File: content/src/content/jcr_root/apps/core/email/components/teaser/v1/teaser/readme.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+* Navigation between container panels via the toolbar
+
+## Technical Details
+
+This component inherit its structure and behavior from the [Teaser component (v2)](https://github.com/adobe/aem-core-wcm-components/tree/main/content/src/content/jcr_root/apps/core/wcm/components/teaser/v2/teaser) of the core.wcm.component package. The changes are the integration of the personalization plugin, which adds a button to insert Adobe Campaign personalization fields in the following locations:
+
+* Configuration dialog toolbar
+* Full screen configuration dialog toolbar
+
+The Teaser also offers the capability to display the image as a background image or as a regular image.
```
Comment (bohnertchris) [inline content/src/content/jcr_root/apps/core/email/components/teaser/v1/teaser/readme.md:35]: How does this work? In the UI, there is no indication of this feature. If it is considered "advanced" or only for developers, we need to document it more clearly here. If It is expected that the author user can use this feature, we will need to include it in the Experience League docs as well.

**Review by bohnertchris — COMMENTED** [review]
If we are going to leave the Authors section at the end of every readme blank, we can just remove them. I added them from the WCM Core Components readmes to be consistent, but if we don't have that information, we can just omit it.

File: content/src/content/jcr_root/apps/core/email/components/image/v1/image/readme.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+The `ImageIT` class is the Selenium test class. It verifies that the image renders correctly when a fixed width is selected and when the scale is set to full size. In both cases, it checks that the `src` HTML attribute of the image contains an absolute URL.
+
+## Information
+
+* **Vendor**: Adobe
+* **Version**: v1
+* **Compatibility**: AEM 6.5
+* **Status**: production-ready
+* **User Documentation**: [https://www.adobe.com/go/aem_cmp_email_button_v1](https://www.adobe.com/go/aem_cmp_email_button_v1)
```
Comment (bohnertchris) [inline content/src/content/jcr_root/apps/core/email/components/image/v1/image/readme.md:54]: Copy/paste error on my part. Sorry.

https://www.adobe.com/go/aem_cmp_email_image_v1

File: content/src/content/jcr_root/apps/core/email/components/teaser/v1/teaser/readme.md (hunk 3 above)
Comment (Buuhuu) [reply content/src/content/jcr_root/apps/core/email/components/teaser/v1/teaser/readme.md:34]: removed

File: content/src/content/jcr_root/apps/core/email/components/image/v1/image/readme.md (hunk 2 above)
Comment (Buuhuu) [reply content/src/content/jcr_root/apps/core/email/components/image/v1/image/readme.md]: removed

**Review by bpauli — APPROVED** [review]

**Pull request merged**